packages/perseus/src/widgets/interactive-graphs/graphs/components/movable-point.tsx

```diff
@@ -33,37 +33,67 @@ export const StyledMovablePoint = (props: Props) => {
 
     const [[x, y]] = useTransform(point);
 
+    const [xMin, xMax] = state.range[0];
+    const [yMin, yMax] = state.range[1];
+
+    const [[verticalStartX]] = useTransform([xMin, 0]);
+    const [[verticalEndX]] = useTransform([xMax, 0]);
+    const [[_, horizontalStartY]] = useTransform([0, yMin]);
+    const [[__, horizontalEndY]] = useTransform([0, yMax]);
+    const showHairlines = dragging && state.showHairlines;
+
     return (
-        <g
-            ref={hitboxRef}
-            className="movable-point"
-            tabIndex={0}
-            style={
-                {
-                    cursor: dragging ? "grabbing" : "grab",
-                    touchAction: "none",
-                    outline: "none",
-                    "--movable-point-color": color,
-                } as any
-            }
-            data-test-id="movable-point"
-        >
-            <circle
-                className="movable-point-hitbox"
-                r={hitboxSizePx / 2}
-                cx={x}
-                cy={y}
-            />
-            <circle className="movable-point-halo" cx={x} cy={y} />
-            <circle className="movable-point-ring" cx={x} cy={y} />
-            <circle className="movable-point-focus-outline" cx={x} cy={y} />
-            <circle
-                className="movable-point-center"
-                cx={x}
-                cy={y}
-                style={{fill: color}}
-                data-test-id="movable-point__center"
-            />
-        </g>
+        <>
+            {showHairlines && (
+                <g>
+                    <line
+                        x1={verticalStartX}
+                        y1={y}
+                        x2={verticalEndX}
+                        y2={y}
+                        stroke={WBColor.green}
```

Comment:
This green looks worse than the original green, but this is the green WB provides and I think we're trying to stick to those colors.